### PR TITLE
Add make content pack cmd

### PR DESCRIPTION
**Pull request by blink-daniel**

### CODE REVIEW

File: cmd/contentpack/main.go [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+	return err
+}
+
+func calcSha2(inputPath string) (string, error) {
+	f, err := os.Open(inputPath)
+	if err != nil {
+		return "", errors.Wrapf(err, "open zip %s content pack", inputPath)
+	}
+	defer f.Close()
```
Comment (ozhey) [inline cmd/contentpack/main.go:125]: Better not to ignore errors, we can create a helper function like this and reuse it everywhere in this file
```
func close(elem io.Closer) {
	err := elem.Close()
	if err != nil {
		log.Warnf("failed to close file: %v", err)
	}
}
```

**Comment by ozhey** [comment]
One more thing - let's export everything that's not in the main func to a package outside the /cmd dir

File: cmd/contentpack/main.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+	return nil
+}
+
+func mkDirIfNotExist(path string) {
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+		err = os.MkdirAll(path, os.ModePerm)
+		if err != nil {
+			log.Fatalf("Failed to create driectory, %s", err)
+		}
```
[the remark this answers is not in the document]
Comment (blink-daniel) [reply cmd/contentpack/main.go:189]: we do, missed that when relaced log.Fatal with errors.Wrap for "leaf" errors handling

File: cmd/contentpack/main.go [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+		_, err = io.Copy(itemWriter, f)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+
+	if err != nil {
+		log.Fatalf("generate zipped content packs: %s", err)
```
[the remark this answers is not in the document]
Comment (blink-daniel) [reply cmd/contentpack/main.go:178]: yeah, missed that when relaced log.Fatal with errors.Wrap for "leaf" errors handling

File: Makefile [hunk 4]
```diff
@@ -6,5 +6,8 @@ image:
 named-actions:
 	go build -o ./bin/ ./cmd/named_actions && ./bin/named_actions --path ${NAMED_ACTIONS_PATH}
 
+content-pack:
+	go run ./cmd/contentpack $(args)
+
```
[the remark this answers is not in the document]
Comment (blink-daniel) [reply Makefile:11]: sure, will align

File: cmd/contentpack/main.go (hunk 1 above)
Comment (blink-daniel) [reply cmd/contentpack/main.go:125]: nice, will call it closeD (D for deferred). log.Warnf introduced in later golang version AFAIK so I'll need to update our min. supported golang for this in go.mod

Comment (ozhey) [reply cmd/contentpack/main.go:125]: Okay cool
And you can just change the standard log library "log" to `log "github.com/sirupsen/logrus"
`
We use it almost everywhere and it supports a few more things

**Comment by blink-daniel** [comment]
need to update docker file to find internal packages

**Review by ozhey — APPROVED** [review]

**Pull request merged**